Allow dependabot PRs to pass by default

```
Error: Cargo.toml version (v0.8.10) was not bumped even though files determined by rust and release wildcards have changed.
Error: Process completed with exit code 1.
```

this was one of the strong suits of the label-based version bump system

--- a/.github/actions/release/action/action.yml
+++ b/.github/actions/release/action/action.yml
@@ -49,8 +49,6 @@ runs:
         committer: ${{ github.actor }}
         message: |
           ${{ github.event.repository.name }} ${{ inputs.version }}
-
-          https://github.com/${{ github.repository }}/blob/${{ github.sha }}/.github/workflows/release.yml
     - name: Tag current commit
       working-directory: release-action
       shell: sh

--- a/.github/actions/release/homebrew-tap/action.yml
+++ b/.github/actions/release/homebrew-tap/action.yml
@@ -40,8 +40,6 @@ runs:
         committer: ${{ github.actor }}
         message: |
           ${{ github.event.repository.name }} ${{ inputs.version }}
-
-          https://github.com/${{ github.repository }}/blob/${{ github.sha }}/.github/workflows/release.yml
     - if: always()
       name: Cleanup
       shell: sh

--- a/.github/actions/release/nur-packages/action.yml
+++ b/.github/actions/release/nur-packages/action.yml
@@ -45,8 +45,6 @@ runs:
         committer: ${{ github.actor }}
         message: |
           ${{ github.event.repository.name }} ${{ inputs.version }}
-
-          https://github.com/${{ github.repository }}/blob/${{ github.sha }}/.github/workflows/release.yml
     - if: always()
       name: Cleanup
       shell: sh

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -23,3 +23,8 @@ updates:
       github-actions:
         patterns:
           - "*"
+    ignore:
+      - dependency-name: "anttiharju/*"
+        update-types:
+          - "version-update:semver-minor"
+          - "version-update:semver-patch"

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -13,6 +13,16 @@ updates:
       cargo:
         patterns:
           - "*"
+  - package-ecosystem: "nix"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    cooldown:
+      default-days: 7
+    groups:
+      nix:
+        patterns:
+          - "*"
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,5 +1,5 @@
 [package]
-version = "0.8.10"
+version = "0.8.11"
 
 license = "MIT"
 description = "Reimplementation of GitHub file paths pattern matcher" # https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#patterns-to-match-file-paths

--- a/lefthook.yml
+++ b/lefthook.yml
@@ -17,6 +17,9 @@ pre-commit:
     - glob: "{.cargo/*,src/*,tests/*,Cargo.lock,Cargo.toml}"
       run: cargo build --all-features
 
+    - glob: Cargo.toml
+      run: cargo update --package compare-changes && git add Cargo.lock
+
     - glob: "*.rs"
       run: cargo fmt
       stage_fixed: true

